### PR TITLE
Convert logging, tracing to annotation-based Prometheus scraping

### PR DESCRIPTION
**Pull request by skhalash**
**Description**

Changes proposed in this pull request:

- Convert loki, jaeger, jaeger-operator from service monitor-based to annotation-based Prometheus scraping
- Remove loki alerting rules

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/11300

### CODE REVIEW

**Review by NHingerl — APPROVED** [review]

File: docs/assets/2.11-2.12-cleanup-servicemonitors.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@ kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-controller
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-nats
 kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com eventing-publisher-proxy
+kubectl delete -n kyma-system --ignore-not-found=true servicemonitors.monitoring.coreos.com logging-loki
```
Comment (chrkl) [inline docs/assets/2.11-2.12-cleanup-servicemonitors.sh:7]: Add tracing ServiceMonitor and Loki Rules to cleanup script.

**Review by NHingerl — APPROVED** [review]

File: docs/assets/2.11-2.12-cleanup-servicemonitors.sh (hunk 1 above)
Comment (skhalash) [reply docs/assets/2.11-2.12-cleanup-servicemonitors.sh:7]: Good point

**Review by NHingerl — APPROVED** [review]

**Review by chrkl — APPROVED** [review]

**Pull request merged**